### ISSUE TITLE
Setting platform to 64 bit does not really set the environment to 64 bit

### ISSUE THREAD

**Issue by bwarren18** [aspnet/AspNetCore.Docs#16489]
I appreciate the additional information but this is still really confusing.

I am trying to use a cmdlet that requires 64 bit but I am struggling to get it to work in Azure AppService.

I created a new AppService, chose a Windows Plan and chose the S1 Tier. 
I tried with the Stack set to .NET Core and .NET (V4.7) and set the Platform to 64 bit.
However in both cases [Environment]::Is64BitProcess returns false and the cmdlet (invoke-sqlcmd) fails.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2eab542-a420-d6c5-d314-a420837cf602
* Version Independent ID: 48c12168-54f3-0b83-f136-565f483a03e9
* Content: [Deploy ASP.NET Core apps to Azure App Service](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-apps/?view=aspnetcore-2.2&tabs=visual-studio)
* Content Source: [aspnetcore/host-and-deploy/azure-apps/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/azure-apps/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

**Comment by guardrex** [comment]
@bwarren18 The SDK is still only 32-bit AFAIK. They have a plan to change that, but they haven't done so yet AFAIK. Does your cmdlet call SDK commands? If so, that probably explains the problem.

**Comment by bwarren18** [comment]
@guardrex, how about just running [Environment]::Is64BitProcess  that doesn't work either if the platform is set to 64-bit.

**Comment by guardrex** [comment]
The switch in the UI to 64-bit affects the IIS w3wp.exe process, not the process for the shell.

There are some similar questions out there pertaining to the shell. I found one at https://github.com/Azure/azure-functions-powershell-worker/issues/232#issuecomment-495405952. If the shell is 32-bit (seems likely here), then the behavior makes sense.

I don't recall ever having to run a PS script in the Azure Apps Kudu PowerShell shell. I almost always run PS for local tasks on my dev machine, so I don't have personal experience with this scenario. I took a stab at getting that module installed in the shell in Azure, but it choked on admin permissions.

The best course is to contact devs on a support forum/chat who use that shell and know its ins-and-outs and/or continue to research for issues/blogs/MS docs that speak to bitness issues in that shell.
